app/src/main/kotlin/cz/covid19cz/erouska/crypto/keystore.kt

```diff
@@ -0,0 +1,93 @@
+package cz.covid19cz.erouska.crypto
+
+import android.content.Context
+import android.security.KeyPairGeneratorSpec
+import cz.covid19cz.erouska.db.SharedPrefsRepository
+import java.math.BigInteger
+import java.nio.charset.Charset
+import java.security.*
+import java.util.*
+import javax.crypto.Cipher
+import javax.security.auth.x500.X500Principal
+
+private const val ENCRYPTION_ALGORITHM = "RSA/ECB/PKCS1Padding"
```

Comment:
We should handle case when device doesn't have encryption algorithm (can happen, it's manufacturer dependent)

Reply:
My goal is to not encrypt the key if crypto fails. Then the DB is still encrypted, but the key is accessible.